myth

```diff
@@ -107,59 +106,53 @@ except KeyError:
 
 # Detect unsupported combinations of command line args
 
-if (args.dynld) and not (args.address):
+if args.dynld and not args.address:
     exitWithError(args.outform, "Dynamic loader can be used in on-chain analysis mode only (-a).")
 
-# Initialize data directry and signature database
+# Initialize data directory and signature database
 
 if not os.path.exists(mythril_dir):
     logging.info("Creating mythril data directory")
-
     os.mkdir(mythril_dir)
 
 # If no function signature file exists, create it. Function signatures from Solidity source code are added automatically.
 
 signatures_file = os.path.join(mythril_dir, 'signatures.json')
 
+sigs = {}
 if not os.path.exists(signatures_file):
-    logging.info("No signature database found. Creating empty database: " + signatures_file + "\n"
-        "Consider replacing it with the pre-initialized database at "
-        "https://raw.githubusercontent.com/ConsenSys/mythril/master/signatures.json")
-
-    sigs = {}
-
+    logging.info("No signature database found. Creating empty database: " + signatures_file + "\n" +
+                 "Consider replacing it with the pre-initialized database at https://raw.githubusercontent.com/ConsenSys/mythril/master/signatures.json")
     with open(signatures_file, 'a') as f:
         json.dump({}, f)
 
-else:
```

Comment:
This `else` is not needed because if that signature file is not exist, the `sigs` will still be empty after reading it from file again.